ksqldb-engine/src/test/java/io/confluent/ksql/function/udaf/offset/EarliestByOffsetUdafTest.java

```diff
@@ -108,6 +194,26 @@ public void shouldComputeEarliestLong() {
     // Then:
     assertThat(res.getInt64(VAL_FIELD), is(321L));
   }
+  
+  @Test
+  public void shouldComputeEarliest2Longs() {
+    // Given:
+    final Udaf<Long, List<Struct>, List<Long>> udaf = EarliestByOffset.earliestLong(2);
```

Comment:
Consider changing the value of `earlestN`, so not all tests use `2`.  Alternate between 1, 2 & 3, maybe?